junit-jupiter-engine/junit-jupiter-engine.gradle.kts

```diff
@@ -13,6 +13,7 @@ dependencies {
 	api(platform(projects.junitBom))
 	api(projects.junitPlatformEngine)
 	api(projects.junitJupiterApi)
+	api(projects.junitJupiterParams)
```

Comment:
Looking for Gradle/Module System help 🙏

What else am I missing to solve the following? Triggered by `junit-jupiter-engine:compileModule`:

```
> Task :junit-jupiter-engine:compileModule FAILED
/Users/stefano/Projects/junit5/junit-jupiter-engine/build/module/org.junit.jupiter.engine/module-info.java:23: error: module not found: org.junit.jupiter.params.converter
	requires org.junit.jupiter.params.converter;
	                                 ^
1 error
```

Reply:
Maybe I could summon @sormuras for a hint? 😇